tests/helpers/order-sample.js

```diff
@@ -0,0 +1,55 @@
+export default function () {
```

Comment:
Why change it to JS?

If keeping it as JS we can also export it as an object `export default { ... }`